doc/guide.rst

```diff
@@ -1,17 +1,265 @@
 Getting started
 ===============
 
-1. Step one: :doc:`install Universum <install>`. Use ``{python} -m universum --help`` to make sure the installation
-   was successful, and also to get the list of available :doc:`parameters <args>`.
+This guide will provide an example step-by-step tutorial on setting up CI for a GitHub project, using Jenkins server,
+GitHub application and Pylint analyzer. These are the steps to take:
 
-2. Step two: :ref:`initialize Universum <additional_commandst#init>` by creating a default :doc:`configuration
-   file <configuring>` and modifying it according to the project needs.
+1. :ref:`Install Universum <guide#install>`
+2. :ref:`Load project to VCS <guide#vcs>`
+3. :ref:`Initialize Unviersum <guide#init>`
+4. :ref:`Configure project <guide#configure>`
+5. :ref:`Add static analyzer <guide#analyzer>`
 
-3. Step three: use ``{python} -m universum run`` to :ref:`check the provided configuration
-   locally <additional_commandst#run>`.
 
-4. Step four: submit a working configuration file to a VCS and pass required :doc:`parameters <args>` to `Universum`
-   to work with it.
+.. _guide#install:
 
-5. Configure CI, using `Universum`, on a CI server. See the following guides for :doc:`TeamCity <teamcity>` or
-   :ref:`GitHub <github_handler#jenkins>`.
+Install Universum
+-----------------
+
+First, before setting up Continious Integration, let's implement and test Universum support locally.
+
+1. Make sure your system meets Universum :doc:`prerequisites <install>`
+2. Install Univesrum using ``{pip} install -U universum`` command from command line
+3. Run ``{python} -m universum --help`` to make sure the installation was successful
+
+If nothing went wrong, you should get a list of available :doc:`command line parameters <args>`.
+
+
+.. _guide#vcs:
+
+Load project to VCS
+-------------------
+
+Using `a series of tutorials on creating new projects
+ <https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/quickstart>`__, let's
+`create a repo <https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/create-a-repo>`__
+on GitHub. Presume we named it ``universum-test-project``. After creating a repo online, clone it locally::
+
+    git clone https://github.com/user/universum-test-project.git
```

Comment:
I think it would be better to follow the common convention of noting example urls and repos.
Github itself uses the following:

 ```
 git clone https://github.com/YOUR-USERNAME/Spoon-Knife
 
 > Cloning into `Spoon-Knife`...
 > remote: Counting objects: 10, done.
 > remote: Compressing objects: 100% (8/8), done.
 > remove: Total 10 (delta 1), reused 10 (delta 1)
 > Unpacking objects: 100% (10/10), done.
 
 ```
Note they write YOUR-USERNAME in italic.